dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/CommunityConverter.java

```diff
@@ -20,14 +29,48 @@
 @Component
 public class CommunityConverter
 		extends DSpaceObjectConverter<org.dspace.content.Community, org.dspace.app.rest.model.CommunityRest> {
+
+	private static final Logger log = Logger.getLogger(CommunityConverter.class);
+	@Autowired(required = true)
+	private CollectionConverter collectionConverter;
+
 	@Override
 	public org.dspace.content.Community toModel(org.dspace.app.rest.model.CommunityRest obj) {
 		return (org.dspace.content.Community) super.toModel(obj);
 	}
 
 	@Override
-	public CommunityRest fromModel(org.dspace.content.Community obj) {
-		return (CommunityRest) super.fromModel(obj);
+	public CommunityRest fromModel(org.dspace.content.Community obj, String projection) {
+		CommunityRest communityRest = super.fromModel(obj, projection);
+		try {
+			for (Community c: obj.getParentCommunities()) {
+				communityRest.setParentCommunity(fromModel(c));
+			}
+		} catch (Exception e) {
+			log.error("Error setting parent community for community "+communityRest.getHandle(), e);
+		}
+
+		if (checkProjection(projection, DSpaceObjectRest.PRJ_CONTEXT)) {
```

Comment:
@tomdesair , I like this idea.  Does it make sense to retain the projection as a String object and allow the ProjectionApplier to interpret that String?